### ISSUE TITLE
PerUnitStorageThroughput included in YAML when adding scratch filesystem

### ISSUE THREAD

**Issue by simone-pignotti** [aws/aws-parallelcluster-ui#116]
## Description
When adding FSX storage of type `SCRATCH_2`, the following line gets automatically added to the YAML configuration file and makes the creation/update fail:
```
[...]
SharedStorage:
  - Name: FsxLustre2
    StorageType: FsxLustre
    MountDir: /scratch
    FsxLustreSettings:
      StorageCapacity: '1200'
      DeploymentType: SCRATCH_2
      PerUnitStorageThroughput: 125
```

## Steps to reproduce the issue
Add a `SCRATCH_2` FsxLustre storage resource to a cluster through the UI.

## Expected behaviour
The `PerUnitStorageThroughput` key should be omitted from the YAML.

## Actual behaviour
The creation fails with the following error message:
```
FsxPersistentOptionsValidator: Per unit storage throughput can only be used when deployment type is one of ['PERSISTENT_1', 'PERSISTENT_2'].
FsxStorageTypeOptionsValidator: 'SCRATCH_2'
```
NB: this also happens any time the same cluster is edited, even if no change to the storage is made. I have to manually delete the `PerUnitStorageThroughput: 125` line at every update.

## Required info
- Region where ParallelCluster UI is installed: eu-west-3
- Version of ParallelCluster UI and ParallelCluster: 3.5.0

**Comment by BarcoMasile** [comment]
Hi Simone,
thanks for reaching out and definitely good catch!
We'll track this bug and fit it in our board ASAP

I'll update this issue when the fix is merged

**Comment by mtfranchetto** [comment]
Hello @simone-pignotti, we have fixed the issue with #148. At the moment, since the code is not released yet, you can fix the issue by manually updating the PCUI Lambda function with the `build_and_update_lambda.sh` script.
